source/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ set(MOORDYN_HEADERS
     MoorDyn.h
     Rod.h
     Waves.h
+    Seafloor.hpp
```

Comment:
I suppose the answer is no... But would it be convenient to add an .h file with C functions, so the users can at least read the bathymetry? (Maybe even edit it)

Reply:
As in, include the bathymetry features in the C API? I don't _think_ that should be too inconvenient. Will have to take a look at the conventions for setting up those headers though, as I've pretty much only used the c++ updated version.

Reply:
Done.